starboard/nplb/time_zone_get_name_test.cc

```diff
@@ -68,6 +68,7 @@ TEST(SbTimeZoneGetNameTest, IsIANAFormat) {
                      // TODO(b/304351956): Remove Etc after fixing builders.
                      std::string("Indian"), std::string("Etc")));
   char* city = strtok(NULL, "/");
+  GTEST_ASSERT_NE(city, (char*)NULL);
```

Comment:
Can we just use ASSERT_NE (instead of GTEST_ASSERT_NE) ?

Reply:
Yes we can. See the last commit